ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -2,29 +2,189 @@
 Tests for ocw_data_parser
 """
 import json
-import unittest
+import pytest
+import os
+import shutil
+import responses
+import boto3
+from moto import mock_s3
+from tempfile import TemporaryDirectory
+from ocw_data_parser.ocw_data_parser import CustomHTMLParser, OCWParser
+import ocw_data_parser.test_constants as constants
 
-from ocw_data_parser import OCWParser
+"""
+Tests for OCW Parser
+"""
+
+@pytest.fixture(autouse=True, scope="session")
+def s3_bucket():
+    with mock_s3():
+        conn = boto3.client("s3",
+                            aws_access_key_id="testing",
+                            aws_secret_access_key="testing")
+        conn.create_bucket(Bucket="testing")
+        responses.add_passthru("https://")
+        responses.add_passthru("http://")
+        s3 = boto3.resource("s3",
+                            aws_access_key_id="testing",
+                            aws_secret_access_key="testing")
+        s3_bucket = s3.Bucket(name="testing")
+        yield s3_bucket
+
+@pytest.fixture(autouse=True, scope="function")
+def ocw_parser():
```

Comment:
Can you create `conftest.py` and move these fixture functions there? pytest will automatically read from that file so you won't need to import the functions, and it will be used in `utils_test.py` too.